src/main/java/com/stripe/model/ChargeRefundCollectionDeserializer.java

```diff
@@ -18,6 +20,7 @@ public ChargeRefundCollection deserialize(JsonElement json, Type typeOfT, JsonDe
 		throws JsonParseException {
 		Gson gson = new GsonBuilder()
 			.setFieldNamingPolicy(FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES)
+			.registerTypeAdapter(ExpandableField.class, new ExpandableFieldDeserializer())
```

Comment:
I had to add the ExpandableField deserializer to both `ChargeRefundCollectionDeserializer` and `FeeRefundColelctionDeserializer`. There's probably a better way of doing this but it would require some refactoring.